draft-quic-atsss-reqs.mkd

```diff
@@ -463,11 +463,21 @@ Network input can be especially beneficial for cases such as:
 
 This section elaborates how non-TCP traffic (UDP or a subset like QUIC) can to be encapsulated into a tunnel between the UE and the ATSSS UPF to enable so called ATSSS higher-layer steering for all traffic, not only TCP or QUIC. This section discusses to what extent QUIC can be used as a tunneling protocol for the ATSSS service and whether gaps are found.
 
+When tunneling non-TCP (e.g., UDP, plain IP) over QUIC the Unreliable Datagram Extension
+{{I-D.ietf-quic-datagram}} can be used. Each data packet would be transported unreliably as a datagram
+over a QUIC connection. Transporting these datagrams unreliably as
+in IPsec or DTLS-based tunnels is especially important for flows that do not require reliable delivery 
+and suffer from unnecessary delay when reliability is added. QUIC datagrams are congestion-controlled, 
+however, as the latency between the UE and the ATSSS proxy is small compared to the end-to-end latency,
+having another local congestion control loops should not impact the end-to-end congestion control negatively.
+
```

Comment:
This is quite speculative. What the end-to-end latency is can vary and is in general not known. I would not claim that the cc control loops will not interact.